### PR TITLE
ct: Add dark mode support to coverage pages

### PR DESCRIPTION
**Pull request by jchristgit**
When common test is tasked with generating coverage via `-cover`, invert the colours of the webpage when the user prefers a dark color scheme. Similar support was added to the other CT-generated pages in commit bbe14c9ff74d7f87d74622f7918e6a8d53963a4d.

## Screenshots

image

image

### CODE REVIEW

**Comment by u3s** [comment]
have you checked Edge browser? can you check it?
my instance renders background as white ...

image

**Comment by jchristgit** [comment]
Sorry for the delay with getting back to you on my PRs, life has been
very busy lately. I will try to test this with Edge and investigate this
by next week. I hope you have an enjoyable Easter weekend :-)

**Comment by u3s** [comment]
> Sorry for the delay with getting back to you on my PRs, life has been very busy lately. I will try to test this with Edge and investigate this by next week. I hope you have an enjoyable Easter weekend :-)

great thanks!

**Comment by jchristgit** [comment]
I rebased from master and reproduced the issue on Edge.

The issue seems to be that Edge needs to have an explicitly set
background color to be able to rotate it. The new changes work for me on
Firefox and Edge.

Whilst checking the PR I also noticed the individual coverage pages
didn't include any dark theme, so I'm wondering whether I forgot to push
something there. I've added a snippet I believe to work, but
unfortunately after rebasing from main I now get a segfault when
compiling, even after cleaning :( I think it's due to my (very wonky)
setup right now, so I'm going to convert this to a draft and get back to
you when I have access to my proper dev machine again.

File: lib/tools/priv/styles.css [hunk 1]
```diff
@@ -94,3 +94,17 @@ td.source {
   white-space: pre;
   font: 12px monospace;
 }
+
```
Comment (u3s) [inline lib/tools/priv/styles.css:97]: is adjustment in this file needed for this PR functionality?

**Comment by u3s** [comment]
> @u3s I got an e-mail about a comment you made on the tools CSS but I think you deleted it, did the question solve itself or should I change something?

no. I think I've made it to quickly and answered it to myself later.
thanks for contributions! sorry for delays.

**Pull request merged**